drivers/wifi/nrf700x/osal/hw_if/hal/inc/fw/phy_rf_params.h

```diff
@@ -16,7 +16,9 @@
 #define NRF_WIFI_RF_PARAMS_CONF_SIZE 42
 
 #ifdef CONFIG_NRF700X_RADIO_TEST
-#define NRF_WIFI_DEF_RF_PARAMS "0000000000002A00000000030303036060606060606060600000000050EC000000000000000000000000007077003F032424001000002800323500000C0008087D8105010071630300EED501001F6F00003B350100F52E0000E35E0000B7B6000066EFFEFFB5F60000896200007A840200E28FFCFF080808080408120100000000A1A10178000000080050003B020726181818181A120A140E0600"
+#define NRF_WIFI_DEF_RF_PARAMS "0000000000002A00000000030303035440403838383838380000000050EC000000000000000000000000007077003F032424001000002800323500000C0008087D8105010071630300EED501001F6F00003B350100F52E0000E35E0000B7B6000066EFFEFFB5F60000896200007A840200E28FFCFF080808080408120100000000A1A10178000000080050003B020726181818181A120A140E0600"
+#define MAX_TX_POWER_FOR_SYSTEM_TEST 30
+#define MAX_TX_POWER_FOR_RADIO_TEST 24
```

Comment:
Try to use short(er) forms wherever possible for macros (while keeping in mind that it does not make things too unreadable :)). E.g. these could be changed to:
`#define MAX_TX_PWR_SYS_TEST 30
#define MAX_TX_PWR_RADIO_TEST 24`
This might help in keeping the code lines below the limit of 100 chars in some cases where these are used.